packages/cli/src/command.js

```diff
@@ -9,7 +9,7 @@ export default class NuxtCommand {
     this.name = name || ''
     this.description = description || ''
     this.usage = usage || ''
-    this.options = Object.assign({}, options)
+    this.options = Object.assign({}, options || {})
```

Comment:
According to the ECMA specs, we don't need this :D (null and undefined are allowed and will be ignored)

![image](https://user-images.githubusercontent.com/5158436/47840997-63f78480-ddcd-11e8-8b91-1bab0c684862.png)

Reply:
@pi0 coolio, will update soon :)